third_party/pcsc-lite/src/Makefile.in

```diff
@@ -160,7 +160,7 @@ DIST_SUBDIRS = $(SUBDIRS)
 am__DIST_COMMON = $(srcdir)/Makefile.in $(srcdir)/config.h.in \
 	$(top_srcdir)/src/PCSC/pcsclite.h.in AUTHORS COPYING ChangeLog \
 	INSTALL NEWS README TODO ar-lib compile config.guess \
-	config.sub depcomp install-sh ltmain.sh missing ylwrap
+	config.sub install-sh ltmain.sh missing ylwrap
```

Comment:
Looks like unrelated change? (here and in `ltmain.sh`)

Reply:
Thanks!